sensor/kubernetes/orchestrator/kubernetes.go

```diff
@@ -32,11 +32,10 @@ func (k *kubernetesOrchestrator) GetNodeScrapeConfig(nodeName string) (*orchestr
 		return nil, err
 	}
 
-	_, hasMasterNodeLabel := node.GetLabels()["node-role.kubernetes.io/master"]
 	_, hasControlPlaneNodeLabel := node.GetLabels()["node-role.kubernetes.io/control-plane"]
 
 	return &orchestrator.NodeScrapeConfig{
 		ContainerRuntimeVersion: node.Status.NodeInfo.ContainerRuntimeVersion,
-		IsMasterNode:            hasMasterNodeLabel || hasControlPlaneNodeLabel,
+		IsMasterNode:            hasControlPlaneNodeLabel,
```

Comment:
Probably outside the scope of this PR, but perhaps worth considering:
```suggestion
		IsControlPlaneNode:      hasControlPlaneNodeLabel,
```

Reply:
I have created ROX-29463 for tracking this.